neuromation/cli/formatters/jobs.py

```diff
@@ -229,6 +232,7 @@ def __init__(self, width: int, username: str):
             "when": [4, 15],
             "image": [5, 40],
             "owner": [5, 25],
+            "cluster_name": [7, 15],
```

Comment:
Interesting, do we need to specify a lower bound? Is not it the same as the length of the corresponding header?

Reply:
You are right

Reply:
In any case we will rewrite formatting , then leave as is